src/main/main.cpp

```diff
@@ -48,8 +54,13 @@ int main(int argc, char *argv[])
 
     QGuiApplication app(argc, argv);
 
+    #ifdef Q_OS_MACOS
+    MacOsEventListener evListener;
+    initMacOsEventListener(&evListener);
```

Comment:
@JohnPoison okay, we can to discuss whether we want to change these code style decisions at a later point.